docs/design/Versions and Releases.md

```diff
@@ -28,24 +49,24 @@ Interim releases for development use (that is, intended for developers working o
 
 Issued as needed.
 
-## Minor Versions
+## Patch Versions
 
 Should be easy to do, in order to encourage fixing problems quickly. All unit tests must pass, and all changes must be properly documented.
 
-## Major Versions
+## Minor Versions
 
 These should be done with more care. Not only must all unit tests pass and everything be documented, but release candidates should be produced and known major users of the component should be encouraged to test the release candidates.
```

Comment:
Maybe require that both documentation and unit tests need to be written for the new functionality? The wording that "everything is documented" is too vague.